best-practices/server-side/framework/python/README.md

```diff
@@ -1,5 +1,47 @@
 # Python
 
+Python is an interpreted high-level general-purpose programming language. Its design philosophy emphasizes code readability with its use of significant indentation. Its language constructs as well as its object-oriented approach aim to help programmers write clear, logical code for small and large-scale projects.
+
+Python is dynamically-typed and garbage-collected. It supports multiple programming paradigms, including structured (particularly, procedural), object-oriented and functional programming. It is often described as a "batteries included" language due to its comprehensive standard library.
+
+Official Docs: https://docs.python.org
+
+The language's core philosophy is summarized in the document [The Zen of Python (PEP 20)](https://www.python.org/dev/peps/pep-0020/), which includes aphorisms:
+```
+Beautiful is better than ugly.
+Explicit is better than implicit.
+Simple is better than complex.
+Complex is better than complicated.
+Flat is better than nested.
+Sparse is better than dense.
+Readability counts.
+Special cases aren't special enough to break the rules.
+Although practicality beats purity.
+Errors should never pass silently.
+Unless explicitly silenced.
+In the face of ambiguity, refuse the temptation to guess.
+There should be one– and preferably only one –obvious way to do it.[a]
+Although that way may not be obvious at first unless you're Dutch.
+Now is better than never.
+Although never is often better than right now.[b]
+If the implementation is hard to explain, it's a bad idea.
+If the implementation is easy to explain, it may be a good idea.
+Namespaces are one honking great idea – let's do more of those!
+```
+
+## Learning Python for the first time? 
+* Take a look at the [Beginners Guide](https://wiki.python.org/moin/BeginnersGuide)
+* Official [Python Tutorials](https://docs.python.org/3/tutorial/index.html#tutorial-index)
+
+## Recommendations
+* Use black for automatic code formatting: https://github.com/psf/black, preferably in conjunction with Continous Integration tools. 
+* Write tests with pytest: https://docs.pytest.org/en/latest/
+* All projects should have a requirements.txt , preferably with pinned requirements. You can use `pip freeze > requirements.txt` on an existing project to get the requirement file started.
+
+## Links and Services
+* In the Fresno area? Why not try the [Fresno Python User Group](https://fresnopython.com)? 
+* Codewars [Python excercises](https://www.codewars.com/collections/basic-python).
+
 ## Frameworks for python
 
-- [Django](django/README.md)
\ No newline at end of file
+- [Django](django/README.md)
```

Comment:
Flask is also popular.

Reply:
@amerikan Good suggestion. Maybe we should add a GH Issue/page for it? Do we have any products that use flask that we can reference?

We have the framework referenced here: https://github.com/Shift3/standards-and-practices/blob/main/best-practices/server-side/framework/README.md but not current pages or docs.

Reply:
Added reference via: 9454ae2696493cf2c0e166304f1196e683b003bc